### PR TITLE
TWINT gateway

### PR DESCRIPTION
**Pull request by iamdevice**

### CODE REVIEW

File: openapi/components/schemas/Gateways/GatewayName.yaml [hunk 1]
```diff
@@ -116,6 +116,7 @@ enum:
   - TestProcessor
   - ToditoCash
   - Trustly
+  - TWINT
```
Comment (ashkarpetin) [inline openapi/components/schemas/Gateways/GatewayName.yaml:119]: Add to acquirer list?

**Review by ashkarpetin — DISMISSED** [review]

**Review by vykhrystiuk — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/TWINT.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          - storeUuid
+          - cashRegisterId
+          - publicKey
+          - privateKey
+          - keyPassphrase
+      settings:
+        type: object
+        description: TWINT settings object
+        properties:
+          cashRegisterType:
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/TWINT.yaml:41]: Not required? What is the default value?

Comment (iamdevice) [reply openapi/components/schemas/GatewayAccountConfig/TWINT.yaml:41]: Oh, you are right... missed that. :(
Thank you!

File: openapi/components/schemas/GatewayAccountConfig/TWINT.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+description: TWINT config
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/TWINT.yaml:6]:
```suggestion
      - credentials
      - settings
```
?

**Review by slavcodev — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**